src/d3-utils/join-helpers.ts

```diff
@@ -0,0 +1,55 @@
+import { 
+  TABLE_ROW_HEIGHT,
+} from '../utils/constants'
+
```

Comment:
Docstrings in this file would be great (and more generally for all functions, exported components, etc)

Reply:
https://looker.atlassian.net/browse/MRKTI-473